docs/blog/2023-06-08-thanos-at-aiven.md

```diff
@@ -0,0 +1,114 @@
+
+The community has been critical for us. One example we ran into around this portion of the implementation which we received some help from the other users and maintainers on the CNCF Slack. We also ran into some other issues which would create replication loops causing a crash as well. We addressed this issue by moving to a routing-ingesting receiver topology as suggested by the community. 
+
+## Cost Savings
+
+After concluding the POC we determined that it was feasible and we also found that the cost savings were pretty significant:
+
+### M3DB Cost breakdown
+Service | Instance type | Price | Count | Total price (monthly)
+------- | ------------- | ----- | ----- | ---------------------
+m3db    | n2d-highmem-32 | $1,172.61 | 18 | $21,106.98 
+m3coordinator | n2d-standard-48 | $1,303.73 | 9 | $11,733.57 
+M3 TOTAL | | $32,840.55
+
+### Thanos Cost breakdown
+Service | Instance type | Price | Count | Total price (monthly)
+------- | ------------- | ----- | ----- | ---------------------
+thanosreceiver | n2d-highcpu-96 | $1,924 | 6 | $11,544
+thanosquery | n2d-highcpu-16 | $321.57 | 6 | $1,929
+thanosstore | n2d-highmem-8 | $293.90 | 3 | $881.7
+thanoscompactor | n2d-highmem-8 | $293.90 | 1 | $293.90
+redis | n2d-highmem-4 | $147.45 | 2 | $294.9
+THANOS TOTAL | | $14,942.68
+
+We are also paying roughly 25% for the storage costs. M3DB has a total of 54TB of storage provisioned today, at a cost of $4320 per month. We could house 216TB of storage for the same cost with Thanos. We are currently generating about 750GB per day, which means we can keep almost a year of metrics for the same cost as M3DB. Additionally, we are backing up M3DB which is using 33TB of object storage at a cost of $1,320 per month. With object storage, we have the added cost for the networking, this is around $1800 per month in additional costs. Here are our estimated costs:
+
+* M3: $38480
+
+* Thanos without any historical data in storage: $16,939
+* Thanos with one month retention in object storage: $17,368
+* Thanos with six months retention: $19,703
+* Thanos with a years retention: $22,447
+* Thanos with 2 years retention: $27,955
+* Thanos with 3 years retention: $33,423
```

Comment:
Hm, Is your obj storage cost alone ~18k USD for 3y data? Sounds bit off and expensive 🤔

Reply:
We have a lot of metrics and manage a large fleet. We also use Thanos (m3) for our billing data which is quite detailed at the service level. This is really the costs, but look how it compares to the cost of M3.